airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -153,6 +154,9 @@ def __init__(
         host: str = "http://localhost:9200",
         frontend: str = "localhost:5601",
         index_patterns: str | None = conf.get("elasticsearch", "index_patterns", fallback="_all"),
```

Comment:
```suggestion
        index_patterns: str = conf.get("elasticsearch", "index_patterns"),
```

Might help with typing checks - this can't actually be none.

Reply:
do you mean to suggest for `index_patterns_callable` param? if not then if we remove fallback then it would be breaking change right?

Reply:
No, for `index_patterns`. Fallback in this context is useless, as there is a default in the provider config. Not breaking to remove it here. Most fallback usage is useless these days.